tools/PI/DevHome.PI/ViewModels/BarWindowViewModel.cs

```diff
@@ -124,23 +146,30 @@ public void SwitchLayoutCommand()
         }
     }
 
+    public void UnsnapBarWindow()
+    {
+        _snapHelper.Unsnap();
+        IsSnapped = false;
+    }
+
     [RelayCommand]
-    public void PerformSnapCommand()
+    public void PerformSnap()
     {
         if (IsSnapped)
         {
-            IsSnapped = false;
+            UnsnapBarWindow();
         }
         else
         {
             // First need to be in a Vertical layout
             BarOrientation = Orientation.Vertical;
+            _snapHelper.Snap();
             IsSnapped = true;
         }
     }
 
     [RelayCommand]
-    public void ShowBigWindowCommand()
+    public void ShowBigWindow()
```

Comment:
Does this have a small and medium window as well?  What is a big window?

Reply:
We used to have a medium window, but don't have them anymore. However, I updated the name for code readability to ToggleExpandedContentVisibility.